frontend/src/scenes/settings/project/SessionRecordingSettings.tsx

```diff
@@ -346,165 +347,159 @@ export function ReplayCostControl(): JSX.Element | null {
     const { updateCurrentTeam } = useActions(teamLogic)
     const { currentTeam } = useValues(teamLogic)
     const { hasAvailableFeature } = useValues(userLogic)
-    const { featureFlags } = useValues(enabledFlagsLogic)
-
-    // some organisations have access to this by virtue of being in a flag
-    // other orgs have access by virtue of being on the correct plan
-    // having the flag enabled overrides the plan feature check
-    const flagIsEnabled = featureFlags[FEATURE_FLAGS.SESSION_RECORDING_SAMPLING]
-    const samplingControlFeatureEnabled = flagIsEnabled || hasAvailableFeature(AvailableFeature.SESSION_REPLAY_SAMPLING)
-    const recordingDurationMinimumFeatureEnabled =
-        flagIsEnabled || hasAvailableFeature(AvailableFeature.REPLAY_RECORDING_DURATION_MINIMUM)
-    const featureFlagRecordingFeatureEnabled =
-        flagIsEnabled || hasAvailableFeature(AvailableFeature.REPLAY_FEATURE_FLAG_BASED_RECORDING)
 
-    const canAccessAnyControl =
-        samplingControlFeatureEnabled || recordingDurationMinimumFeatureEnabled || featureFlagRecordingFeatureEnabled
+    const samplingControlFeatureEnabled = hasAvailableFeature(AvailableFeature.SESSION_REPLAY_SAMPLING)
+    const recordingDurationMinimumFeatureEnabled = hasAvailableFeature(
+        AvailableFeature.REPLAY_RECORDING_DURATION_MINIMUM
+    )
 
-    return canAccessAnyControl ? (
-        <>
-            <p>
-                PostHog offers several tools to let you control the number of recordings you collect and which users you
-                collect recordings for.{' '}
-                <Link
-                    to="https://posthog.com/docs/session-replay/how-to-control-which-sessions-you-record"
-                    target="blank"
-                >
-                    Learn more in our docs.
-                </Link>
-            </p>
+    return (
+        <PayGateMini feature={AvailableFeature.SESSION_REPLAY_SAMPLING}>
```

Comment:
I didn't really want to upgrade the paygatemini signature to accept an array as a key so I just gated it on one of the features. It doesn't seem like we are planning to enable the features separately, so I think this is fine.